middleware/middleware-1.3.14.md

```diff
@@ -0,0 +1,80 @@
+---
+slug: /release-notes/middleware/1.3.14
+title: Version 1.3.14
+---
+
+# fiskaltrust.Middleware 1.3.14 (Germany)
```

Comment:
AFAIK this is the first version where we have MySQL as GO Live version, should we add a section?

Reply:
I added a section about that, good point!